src/sgp/Cursor_Control.cc

```diff
@@ -357,3 +362,16 @@ void SetExternMouseCursor(SGPVObject const& vo, UINT16 const region_idx)
 	guiExternVo         = &vo;
 	gusExternVoSubIndex = region_idx;
 }
+
+// Checks if uiCursorIndex is VIDEO_DEFAULT_TO_NO_CURSOR. If so, we parse it as VIDEO_NO_CURSOR or 0 (generic cursor)
+UINT32 ReturnCursorIndex(UINT32 uiCursorIndex)
```

Comment:
I think we can come up with a better name — to me ReturnCursorIndex with a uiCursorIndex sounds like something that will get inlined due to triviality. Maybe just ModifyCursorIndex?

Reply:
I can change it to that. I'll wait for feedback from the other questions, and I'll update PR.